libs/vkd3d/device.c

```diff
@@ -1788,6 +1789,7 @@ static HRESULT d3d12_device_create_vkd3d_queues(struct d3d12_device *device,
                 &queue_info->vk_properties[i], &device->queues[i]))))
             goto out_destroy_queues;
 
+        device->unique_queue_mask |= 1u << i;
```

Comment:
Should the internal compute queue be included here? From what I understand, this mask should only be used for user-visible queues since the memory clear code will do internal compute -> all user submissions.

Reply:
I've not excluded it here since we have to filter out the internal queue anyway. The internal queue *may* still point to a user queue so it doesn't make much of a difference anyway.

Reply:
Changed this regardless.